### DISCUSSION TITLE
Bain group analysis

### DISCUSSION THREAD

**Discussion by NovakEvana** [ORTools4MTEX/ORTools#25] [Ideas]
Hello Frank and contributors of the ORTools,

I'm just wondering if we have the integration for Bain group (martensite block) analysis in our ORTool yet. It would be great if you could have that one beside the CP and individual variant analysis that has already been implemented. Please correct me if you've already enabled that function since I couldn't find it within the current ORTool version. Thanks.

**Comment by frankNiessen** [comment]
Hi Evana, 

We do not have this yet and it is on the to-do list. I will get back to you when we have implemented it. 

Best wishes
Frank

**Reply by NovakEvana** [reply]
Thanks Frank. Just a quick question: do you have a roadmap of what will be implemented in the next version (major changes)? Really looking forward to the new ORTools features.

**Reply by frankNiessen** [reply]
There is no real roadmap as of now, but maybe I will have to sit together with my colleague Azdiar and discuss this with him. In terms of feature implementation there always needs to be a demand, so you have to make yourself heard (as you did), and it needs to overlap with my personal need in research, as most of the development of ORTools is still happening in free time. I can disclose that we will soon release a new feature within MTEX that will become part of ORTools as well which should be interesting for the community of martensitic phase transformations, but I am not sure I am allowed to reveal it yet :).

**Reply by NovakEvana** [reply]
Thanks Frank for sharing. I'll close this thread and wait for the coming features release instead :). Will keep you and the team know if any ideas pop up in my head. 

Thanks Frank and Merry Christmas!

**Comment by AzdiarGazder** [comment]
Hi @NovakEvana,

I'm working with @frankNiessen on the best way to introduce Bain group analysis to Mtex / ORTools. There is working code but we still have to hash a few things out together before releasing it to everyone. As @frankNiessen  mentioned, we do development work in our free time and/or if it coincides with our research demands, so please be patient with us. It'll definitely be there soon! :)
